content/_index.md

```diff
@@ -76,6 +76,9 @@ We are also available on this e-mail: user-cont-team@redhat.com.
   every two weeks. To accomplish this, the release and validation processes are
   completely automated.
 
+* All the tests are passing in CI systems for master branches of every our
+  project. No excuses.
```

Comment:
Isn't this a little bit negative?

Reply:
please suggest better wording

Reply:
` No excuses.` does not match the "serious" tone of the rest of the points...

Reply:
YES!

Reply:
Although, I can remove it :)

So, who wants me to remove it? :P

Reply:
+1 for keeping it like this. I think it is worth highlighting, cause we (me including) have been merging without green CI a lot.

Reply:
OK

We can make merging-via-zuul required to avoid merging PRs without green tests. (And ideally make te zuul check other checks.)

Reply:
> OK
> 
> We can make merging-via-zuul required to avoid merging PRs without green tests. (And ideally, make the zuul check other checks.)

Actually zuul can do such a thing, I was told this at AnsibleFest: you can set up jobs which will require other GitHub checks to be passing (also got examples, hopefully)

Reply:
Is "of every our project" grammatically correct?